web_widget_color/README.rst

```diff
@@ -0,0 +1,92 @@
+Color widget for Odoo web client
+================================
+
+This module aims to add a color picker to Odoo.
+
+It's a `jsColor <http://jscolor.com/>`_ lib integration.
+
+
+Features
+========
+
+* The picker allow the user to quickly select a color on edit mode
+
+  |picker|
+
+  .. note::
+
+      Notice how html code and the background color is updating when selecting a color.
+
+
+* Display the color on form view when you are not editing it
+
+  |formview|
+
+* Display the color on list view to quickly find what's wrong!
+
+  |listview|
+
+
+Usage
+=====
+
+You need to declare a char field::
+
+    color = fields.Char(
+        string="Color",
+        help="Choose your color"
+    )
+
+
+In the view declaration, put widget='color' attribute in the field tag::
+
+    ...
+    <field name="arch" type="xml">
+        <tree string="View name">
+            ...
+            <field name="name"/>
```

Comment:
Remove this useless line that can confuse also. Is it also valid for form. Then it should be mentioned.